server/docker/Dockerfile

```diff
@@ -1,18 +1,24 @@
 FROM alpine:3.8
 MAINTAINER Evgeny Golyshev <eugulixes@gmail.com>
 
-ENV BRANCH master
+ENV BRANCH add-gino
 
 ARG VCS_REF
 
 LABEL org.label-schema.vcs-ref=$VCS_REF \
       org.label-schema.vcs-url="https://github.com/tolstoyevsky/hubot-huntflow-reloaded"
 
 RUN apk --update add \
-    curl \
     bash \
+    curl \
+    gcc \
     git \
-    python3 \
+    musl-dev \
+    postgresql-client \
+    postgresql-dev \
+    python3-dev \
+    sudo \
+
```

Comment:
Remove the blank line. It may cause the warning message and some problems on newer Docker engines.